### ISSUE TITLE
scriptRoot doesn't work (or documentation needs clarifying)

### ISSUE THREAD

**Issue by raintonr** [nokxs/iobroker-javascript-vs-code-extension#74]
**Describe the bug**
No matter what I set `scriptRoot` config item too, download of scripts from ioBroker server puts them in top level project folder.

**To Reproduce**
Change `scriptRoot` to something. It is ignored.

**Expected behavior**
I have the following folder:
```
.../src/myProject
.../src/myProject/someJunk
.../src/myProject/someOtherJunk
.../src/myProject/ioBrokerScripts
```

I want to open folder `.../src/myProject` in vscode and browse around things. In that folder I have a `.iobroker-config.json` file. In that file I have the setting `  "scriptRoot": "iobroker-scripts"`.

According to the documentation when I download from ioBroker the scripts should end up in `.../src/myProject/ioBrokerScripts` but instead, they end up in `.../src/myProject`. I have tried various folders but none seem to work - scripts always end up in top level folder.

**Version information (please complete the following information):**
 - Extension version: v0.13.1
 - VS Code version: 1.63.2
 - ioBroker admin version: v5.2.3
 - ioBroker nodejs version: v12.22.5
 - ioBroker javascript adapter version: 5.2.18

**Comment by nokxs** [comment]
Thanks for the nice description of the bug. I will try to reproduce it, because it's working fine for me. 

From your description it seems you've done everything correct. Would it be possible for you to share your `.iobroker-config.json`?

A config working for me looks like this:

```json
{
  "ioBrokerUrl": "http://iobroker",
  "socketIoPort": 8081,
  "scriptRoot": "iobroker-scripts",
  "adminVersion": "admin5"
}
```

**Comment by nokxs** [comment]
I was able to reproduce the bug. It never happened to me, because I've got all my scripts in sub directories and only scripts in the root directory were affected. 

Please let me know if the just released version 0.13.2 fixes your problem.

**Comment by raintonr** [comment]
Yup - that's better. Thank you for the quick response :smile:
